pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -117,7 +118,9 @@ EOF
     fi
    '';
 
-  qmakeFlags = optional enableProprietaryCodecs "-- -proprietary-codecs";
```

Comment:
This is causing a rebuild on other platforms (I think) because the number of list elements changes. Until we decide to use `-system-ffmpeg` on all platforms, could you change this so that it does not rebuild on other platforms?

Reply:
I've just noticed I also made a mistake in my addition to propagatedBuildInputs, which means ffmpeg gets added on other platforms, when it shouldn't.